src/app/components/navbar/navbar-user-account/navbar-user-account.component.html

```diff
@@ -1,19 +1,21 @@
-<div (click)="openAlertBox()" class="outer-div">
+<div (click)="openAlertBox()" class="outer-div" appClickStopPropagation>
   <img
     ngSrc="assets/icons/chips-icon.svg"
     alt="credits"
     height="26"
     width="40"
-    style="margin-left: 10px"
+    style="margin-left: 0.75rem"
   />
-  <p id="user-credits">{{ userProfile?.coins }}</p>
+  <div id="user-credits">{{ userProfile?.coins }}</div>
   <div id="account-image"></div>
 </div>
 
 <div *ngIf="this.isLoggedIn">
   <app-account-page-user-profile
     *ngIf="showAlertBox"
-    [username]="userProfile?.username?.toString()"
+    [@fadeInOut]=""
```

Comment:
is this necessary?

Reply:
If you're referring to the empty string after the animation, then no, it's not necessary, as in the app works fine without it, but not putting a value there gives an annoying warning: 
![image](https://github.com/HibyteIntern/internal-betting-fe/assets/126521332/4ceced64-e0a5-459c-8a4d-e18b1815bd73)